data/json/recipes/recipe_food.json

```diff
@@ -59,6 +59,44 @@
   ]
 },{
   "type" : "recipe",
+  "result": "salt",
+  "id_suffix": "from_mineral",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_OTHER",
+  "skill_used": "cooking",
+  "difficulty": 4,
+  "skills_required": [ "cooking", 4 ],
+  "time": 90000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [["atomic_survival", 4], ["textbook_chemistry", 5], ["adv_chemistry", 4]],
```

Comment:
4/4 difficulty and booklearn
Pretty sure salt was produced from rock salt even in medieval times.